Update do_kafka_test to set DOCKER_HOST_IP

mwodahl opened the pull request:
Update the do_kafka_test script to automatically determine DOCKER_HOST_IP in the case that it is not set instead of automatically exiting

dmccoystephenson reviewed (changes requested):
When I run the script without setting DOCKER_HOST_IP first, it appears to remain unset and kafka fails to start.

```
root@L0674:/mnt/c/projects/cdot-projects/jpo-cvdp# ./do_kafka_test.sh

Step 1/5: Set up test environment
=== Setup Info ===
DOCKER_HOST_IP:
KAFKA_CONTAINER_NAME: jpo-cvdp-kafka-1
MAP_FILE: data/I_80.edges
BSM_DATA_FILE: data/I_80_test.json
TIM_DATA_FILE: data/I_80_test_TIMS.json
PPM_CONTAINER_NAME: test_ppm_instance
PPM_IMAGE_TAG: do-kafka-test-ppm-image
PPM_IMAGE_NAME: jpo-cvdp_ppm
==================
Removing network jpo-cvdp_default
WARNING: Network jpo-cvdp_default not found.
Creating network "jpo-cvdp_default" with the default driver
Creating jpo-cvdp_zookeeper_1 ... done
Creating jpo-cvdp_kafka_1     ... done

Step 2/5: Wait for Kafka to create topics
Kafka container 'jpo-cvdp-kafka-1' is not running. Exiting.
Stopping jpo-cvdp_zookeeper_1 ... done
Removing jpo-cvdp_kafka_1     ... done
Removing jpo-cvdp_zookeeper_1 ... done
Removing network jpo-cvdp_default
```

I am running this with WSL in my local environment.

mwodahl commented:
Updated the ifconfig command to be compatible with WSL and added an additional check to exit if docker_host_ip cannot be determined after ifconfig is run

dmccoystephenson reviewed (approved):
The DOCKER_HOST_IP environment variables successfully gets set in my WSL terminal now, so this looks good!

As a side note, for some reason Docker is naming the kafka container jpo-cvdp_kafka_1 instead of jpo-cvdp-kafka-1 for me, so I had to modify line 16 to get the script to run. We may want to consider checking for both formats or grabbing the name of the kafka container dynamically. This may be out of scope of this PR, though. I've opened a backlog work item for this.

payneBrandon reviewed (approved):
First run failed to connect, but I think Kafka didn't spin up properly. Subsequent runs look good!
